src/Navigation/Skill/SectionHead.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./index.css";
+function SectionHead(props) {
+  return (
+    <>
+      <div className="skills_profile">
+        <div className="skills_profile_logo"></div>
```

Comment:
For no-child divs, use self-closing tags, it is more neat. Like:
`<div className="skills_profile_logo" />`